openebs/pkg/v1/maya_api.go

```diff
@@ -68,7 +68,7 @@ func (v OpenEBSVolume) GetMayaClusterIP(client kubernetes.Interface) (string, er
 }
 
 // CreateVsm to create the Vsm through a API call to m-apiserver
-func (v OpenEBSVolume) CreateVsm(vname string, size string) (string, error) {
+func (v OpenEBSVolume) CreateVsm(vname, size, storageClassName string) (string, error) {
```

Comment:
The parameters are increasing. Not a good sign. Take this up in some other PR.

Reply:
Sure! I will create a struct to encapsulate those and will pass it along.

Reply:
@AmitKumarDas  created issue for this https://github.com/openebs/openebs/issues/787